calico-cloud/get-started/connect/requirements/system-requirements.mdx

```diff
@@ -36,7 +36,7 @@ kubectl get <type> -n <namespace> <resource-name> -o yaml | grep ' addonmanager.
 
 ### Step 3: Prepare your infrastructure to connect to {{prodname}}
 
-Pods running in your kubernetes cluster should be allowed to connect to the following endpoints:
+Pods on your Kubernetes clusters must allow outbound connections to the following endpoints:
```

Comment:
I think this change doesn't make things clearer. I think the original wording was accurate.